test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageCreateModelExpressionTest.cs

```diff
@@ -268,8 +281,22 @@ public override Task ExecuteAsync()
             }
         }
 
-        private class TestRazorPage : RazorPage<RazorPageCreateModelExpressionModel>
+        private class TestRazorPage : TestRazorPage<RazorPageCreateModelExpressionModel>
         {
+            public ModelExpression CreateModelExpression<TValue>(Expression<Func<RazorPageCreateModelExpressionModel, TValue>> expression)
+            {
+                return ModelExpressionProvider.CreateModelExpression(ViewData, expression);
+            }
```

Comment:
No reason to maintain anything like the old API. Use `page.ModelExpressionProvider` directly in the tests and test pages.